packages/vim_runtime.rb

```diff
@@ -128,7 +125,7 @@ def self.install
         silent! call mkdir(expand(&g:undodir), 'p', 0700)
       endif
     EOF
-    system "sed -i 's/set mouse=a/set mouse-=a/g' #{CREW_DEST_PREFIX}/share/vim/vim82/defaults.vim"
+    system "sed -i 's/set mouse=a/set mouse-=a/g' #{CREW_DEST_PREFIX}/share/vim/vim#{@vim_version}/defaults.vim"
```

Comment:
Is `defaults.vim` accessible with this path?

Reply:
Yes.
```
crew files vim_runtime | grep defaults
/usr/local/share/vim/vim90/defaults.vim
```

Reply:
(The sed succeeds.)